docs/next-steps.md

```diff
@@ -0,0 +1,51 @@
+# Welcome to the Next Steps for your Kubernetes cluster.
+
+This guide is for [Kubernetes Quick Start](https://aws.amazon.com/quickstart/) admins and users. This guide assumes you have deployed the Kubernetes Quick Start by walking through [Amazon’s Quick Start PDF](https://s3.amazonaws.com/quickstart-reference/vmware/latest/doc/vmware-kubernetes-on-the-aws-cloud.pdf), which uses Amazon’s CloudFormation console.
+
+Now that you have your Kubernetes stack on Amazon, we recommend setting up [WordPress with Helm](tutorial-wordpress.md) as a demo application to explore your new cluster.
+
+
+> Note:
+>
+>This stack is appropriate for proof of concept (PoC), experimentation, development, and small internal-facing projects. Consider this a test drive.
+>
+>This stack does not currently support upgrades and must be rebuilt for new versions.
+
+
+## Release notes
+Release notes: AWS Quick Start for Kubernetes by VMware
+
+The Quick Start builds Kubernetes 1.13.2.
+
+## Next steps
+If you’ve completed the Kubernetes Quick Start on AWS, we recommend that you try these next steps:
+
+ - Check that your cluster is configured and working as expected by running [Sonobuoy](https://github.com/heptio/sonobuoy) against your cluster. Sonobuoy provides cluster diagnostics by running Kubernetes conformance tests.
+ - Explore your cluster and deploy a demo application: [WordPress with Helm](tutorial-wordpress.md)
+ - Allow traffic: [Allow outside traffic to your cluster with load balancing](tutorial-traffic.md)
+
+## Additional resources
+ VMware has collected some links to help you explore your Kubernetes cluster:
+
+- Bitnami shows how to deploy a MEAN stack: [Deploy, Scale And Upgrade An Application On Kubernetes With Helm](https://docs.bitnami.com/kubernetes/how-to/deploy-application-kubernetes-helm/)
+ - Bitnami manages a library of [Kubernetes-ready applications](https://kubeapps.com/)
+ - Project Calico goes into detail on networking for Kubernetes: [Calico for Kubernetes](http://docs.projectcalico.org/v2.0/getting-started/kubernetes/)
```

Comment:
Is this still the right link? It currently links to the v2.0 docs for Calico.

Reply:
Yea it should go to the newer version.